src/memphis.ts

```diff
@@ -708,6 +725,14 @@ class Memphis {
     isConnected() {
         return !this.brokerManager.isClosed();
     }
+
+    public setConsumeHandler(
```

Comment:
who should use it?

Reply:
It is used by the @MemphisConsume() decorator to save the handlers to the memphis instance. 

Then after a memphis.connect() connection is successful, the memphis instance will now create consumers and attach those saved handlers. 

This step is necessary because TypeScript evaluates method decorators very early in the build stage. It evaluates and calls them before memphis.connect() gets called. So we need to store the contents of the @MemphisConsume() decorator (and the handler attached to it) in the memphis instance. So that we can attach them after a successful connection.

Reply:
ok so please add the char "_" to the beginning of the function name to indicate it is for internal use

Reply:
I've prefixed it